client/src/FilterButton.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const getType = (type) => {
```

Comment:
filterButton should only contain the component

Reply:
For both functions that need to be moved out, should I put it both on utils, or do I move them over to App?